tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -0,0 +1,149 @@
+              - ip_netmask: 0.0.0.0/0
+                next_hop: 192.168.121.1
+              external_mtu: 1500
+              external_vlan_id: 44
+              external_cidr: '24'
+              external_host_routes: []
+              internal_api_mtu: 1500
+              internal_api_vlan_id: 20
+              internal_api_cidr: '24'
+              internal_api_host_routes: []
+              storage_mtu: 1500
+              storage_vlan_id: 21
+              storage_cidr: '24'
+              storage_host_routes: []
+              tenant_mtu: 1500
+              tenant_vlan_id: 22
+              tenant_cidr: '24'
+              tenant_host_routes: []
+              role_networks:
+              - InternalApi
+              - Storage
+              - Tenant
+              networks_lower:
+                External: external
+                InternalApi: internal_api
+                Storage: storage
+                Tenant: tenant
+              # edpm_nodes_validation
+              edpm_nodes_validation_validate_controllers_icmp: false
+              edpm_nodes_validation_validate_gateway_icmp: false
+
+              edpm_ovn_metadata_agent_DEFAULT_transport_url: rabbit://default_user_-secret@rabbitmq.openstack.svc:5672
+              edpm_ovn_metadata_agent_metadata_agent_ovn_ovn_sb_connection: tcp:172.17.0.31:6642
+              edpm_ovn_metadata_agent_metadata_agent_DEFAULT_nova_metadata_host: 127.0.0.1
+              edpm_ovn_metadata_agent_metadata_agent_DEFAULT_metadata_proxy_shared_secret: 12345678
+              edpm_ovn_metadata_agent_DEFAULT_bind_host: 127.0.0.1
+              edpm_chrony_ntp_servers:
+              - clock.corp.redhat.com
+
+              ctlplane_dns_nameservers:
+              - 192.168.121.1
+              dns_search_domains: []
+              edpm_ovn_dbs:
+              - 172.17.0.31
+
+              edpm_ovn_controller_agent_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-ovn-controller:{{ image_tag }}"
+              edpm_iscsid_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-iscsid:{{ image_tag }}"
+              edpm_logrotate_crond_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-cron:{{ image_tag }}"
+              edpm_nova_compute_container_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-nova-compute:{{ image_tag }}"
+              edpm_nova_libvirt_container_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-nova-libvirt:{{ image_tag }}"
+              edpm_ovn_metadata_agent_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-neutron-metadata-agent-ovn:{{ image_tag }}"
+
+              gather_facts: false
+              enable_debug: false
+              verbosity: 4
+              # edpm firewall, change the allowed CIDR if needed
+              edpm_sshd_configure_firewall: true
+              edpm_sshd_allowed_ranges: ['192.168.122.0/24']
+              # SELinux module
+              edpm_selinux_mode: permissive
```

Comment:
after changing it to permissive, I could successfully run all the AEE jobs from the dataplane-operator:
```
[dpadev@osp-dev-05 director_standalone]$ oc get openstackansibleee | grep adoption
dataplane-deployment-configure-network-edpmadoption     ["ctlplane","internalapi","storage","tenant"]   True     AnsibleExecutionJob complete
dataplane-deployment-configure-openstack-edpmadoption   ["ctlplane","internalapi","storage","tenant"]   True     AnsibleExecutionJob complete
dataplane-deployment-configure-os-edpmadoption          ["ctlplane","internalapi","storage","tenant"]   True     AnsibleExecutionJob complete
dataplane-deployment-install-openstack-edpmadoption     ["ctlplane","internalapi","storage","tenant"]   True     AnsibleExecutionJob complete
dataplane-deployment-install-os-edpmadoption            ["ctlplane","internalapi","storage","tenant"]   True     AnsibleExecutionJob complete
dataplane-deployment-run-openstack-edpmadoption         ["ctlplane","internalapi","storage","tenant"]   True     AnsibleExecutionJob complete
dataplane-deployment-run-os-edpmadoption                ["ctlplane","internalapi","storage","tenant"]   True     AnsibleExecutionJob complete
dataplane-deployment-validate-network-edpmadoption      ["ctlplane","internalapi","storage","tenant"]   True     AnsibleExecutionJob complete
```

Reply:
:+1: